apps/teams-test-app/src/components/LocationAPIs.tsx

```diff
@@ -55,6 +55,7 @@ const ShowLocation = (): React.ReactElement =>
           return 'Completed';
         },
         withCallback: (locationProps, setResult) => {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
>  // eslint-disable-next-line @typescript-eslint/no-unused-vars [](http://example.com/codeflow?start=8&length=63)

Not saying you have to fix it as part of this change, but should we actually be using the status variable in this test app? E.g. putting it into the result somewhere?
(you can always file a bug to fix it separately if so)

Reply:
Good idea. I initially thought no because this function is deprecated (I wasn't sure we wanted to allocate more resources towards something we're not maintaining) and I couldn't find any documentation on what exactly the status means here. (succeeded = true maybe?) But after thinking more on it thanks to your comment ( :smile: ) I realize all I would have to do is add to the string message displayed. I'll be making that change. As for understanding what exactly the status means, I imagine it's probably just true = good and false = something went wrong...?

Reply:
Is there some automated test that is going to fail because the string is different now? I'm assuming if there was it would have failed on your PR with the change... does that mean there's no automation that calls and tests this?

Reply:
This was a great question! (I missed thinking about e2e.) It seems it passes the tests, although I'm not sure that's intended. I looked at the tests in the app host sdk side and I'm not quite sure when it utilizes the callback version here vs the promises version. For now I'm thinking I'll actually revert my latest commit to use the eslint-disable comment and create a ticket as you mentioned we don't necessarily have to fix it in the PR- and then I can grab the ticket and work on that one next. What do you think?

Reply:
Sounds good.